istioctl/pkg/precheck/precheck.go

```diff
@@ -223,6 +224,27 @@ func checkDestinationRuleTLS(cli kube.CLIClient, messages *diag.Messages) error
 	return nil
 }
 
+func checkVirtualServiceHostMatching(cli kube.CLIClient, messages *diag.Messages) error {
+	virtualServices, err := cli.Istio().NetworkingV1alpha3().VirtualServices(metav1.NamespaceAll).List(context.Background(), metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+	for _, vs := range virtualServices.Items {
+		for _, hostname := range vs.Spec.Hosts {
+			if host.Name(hostname).IsWildCarded() {
+				res := ObjectToInstance(vs)
+				messages.Add(msg.NewUnknownUpgradeCompatibility(res,
```

Comment:
I think we want UpdateIncompatibility vs UnknownUpgradeCompatibility. UnknownUpgradeCompatibility means we cannot detect anything; UpdateIncompatibility means the behavior probably changed

Reply:
Got it; will update